dlp/snippets/jobs/jobs_test.go

```diff
@@ -182,6 +186,133 @@ func TestDeleteJob(t *testing.T) {
 	}
 }
 
+func TestCreateJob(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	var buf bytes.Buffer
+	// createBucketForCreatJob will create a bucket and upload a txt file
+	bucketName, err := createBucketForCreatJob(t, tc.ProjectID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	gcsPath := "gs://dlp-go-lang-test/test.txt"
```

Comment:
Tests are failing. Looks like gcsPath is hardcoded and should be based on the bucket just created.

Reply:
- Not sure but as per the error in CI Looks like it's permission related issue, https://source.cloud.google.com/results/invocations/4dbc0e64-6595-4177-ab8b-b0ecbbfcc9cf/targets/github%2Fgolang-samples%2Fdlp%2Fsnippets%2Fjobs/tests 
Still looking at it.

Reply:
Fixed it.